apps/web-giddh/src/app/ledger/ledger.component.ts

```
@@ -3862,6 +3862,24 @@ export class LedgerComponent implements OnInit, OnDestroy {
+    public pageChanged(event: any): void {
+        if (typeof event === 'string') {
+            if (this.isAdvanceSearchImplemented && !this.trxRequest.q?.length) {
+                this.advanceSearchRequest.paginationToken = event;
+                this.getAdvanceSearchTxn();
+            } else {
+                this.trxRequest.paginationToken = event;
+                this.getTransactionData();
+            }
+        }
+    }
+
```

Biome (2.1.2)

[error] Duplicate class member name "pageChanged"



(lint/suspicious/noDuplicateClassMembers)